File: leetcode-algorithms/026. Remove Duplicates from Sorted Array/solution.py
def removeDuplicates(nums):
	"""
	:type nums: List[int]
	:rtype: int
	"""
	if not nums:
		return 0
	res = 1
	for i in range(1, len(nums)):
		if nums[i] != nums[i-1]:
			nums[res] = nums[i]
			res += 1
	return res

if __name__ == '__main__':
	nums = [1, 1, 2]
	res = removeDuplicates(nums)
	print(res,nums)
        

File: leetcode-algorithms/026. Remove Duplicates from Sorted Array/solution.cpp
#include <iostream>
#include <vector>
using namespace std;

int removeDuplicates(vector<int>& nums) {
	if(nums.size()==0)
		return 0;
	int len = 1;
	for(int i=1;i<nums.size();i++)
	{
		if(nums[i]!=nums[i-1])
			nums[len++] = nums[i];
	}
	return len;
}

int main()
{
	vector<int> nums = {1,1,2};
	int res = removeDuplicates(nums);
	cout<<res<<endl;
	return 0;
}